components/rmrk/utils.ts

```diff
@@ -256,7 +256,7 @@ export const getView = (rmrkString: string): RmrkMint | RmrkView | null => {
 
 export const unwrap = (rmrkString: string): any | null => {
```

Comment:
please remove, already implemented in https://github.com/vikiival/minimark/blob/master/src/rmrk/unwrap.ts

Reply:
This function is used in the code. In the Utils.getView() function